pkg/advisory/create.go

```diff
@@ -57,6 +57,12 @@ func Create(req Request, opts CreateOptions) error {
 			return fmt.Errorf("unable to create advisory %q for %q: %w", newAdvisoryID, req.Package, err)
 		}
 
+		// Update the schema version to the latest version.
+		err = documents.Update(v2.NewSchemaVersionSectionUpdater(v2.SchemaVersion))
+		if err != nil {
+			return fmt.Errorf("unable to update schema version for %q: %w", req.Package, err)
+		}
```

Comment:
This stanza is repeated a few times throughout the PR diff. It should be consolidated into one location — and that's not easy to do right now. We're due for a small refactor in how we interact with the underlying data, at which point this will get better.